### ISSUE TITLE
Launch window with no reaction

### ISSUE THREAD

**Issue by BSarmady** [PrismLauncher/PrismLauncher#988]
### Operating System

Windows

### Version of Prism Launcher

Why is this happening?

![image](https://user-images.githubusercontent.com/2040435/231154338-7fb960f8-f6cc-46c4-9683-59a847753f97.png)

It gets stuck at this dialog, and if I don't kill the task it will stay there forever. 

Abort button doesn't have an event, Window close doesn't have effect either and progress bar stays at 0%

### Version of Qt

I'm not sure, 

I just run Prism launcher 7.0-develop

There is no copy feature in the about dialog, otherwise I could paste the commit version, but the font is too small for me to even try and read and enter it here. but build date is 2023-02-09

### Description of bug

There was no cat button in that dialog, but there was one in the main window, when I pressed it, it showed a very surprised mouse!

### Steps to reproduce

Create a shortcut of an instance, close the main window and run the instance using the shortcut.

### Suspected cause

No internet connection?

I still can run the instance from main window by pressing launch button!

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Scrumplex** [comment]
Check if you are affected by #660

**Comment by BSarmady** [comment]
Nope, I still can open main window (with internet or without) and do everything even downloading mods (obviously  when I have internet access)

but when I don't have internet access, clicking instance shortcuts, only opens this window and nothing happens. same instance is still accessible from main application.

for my 4 years old, I only put instance shortcut (not the app) so she do not mess up the instances. And I'm sure you know what happens if a kid's game doesn't run.
